src/openfl/utils/_internal/ShaderMacro.hx

```diff
@@ -199,6 +436,256 @@ class ShaderMacro
 		return fields;
 	}
 
+	private static inline function getDefaultGLVersion():String
+	{
+		// Specify the default glVersion.
+		// We can use compile defines to guess the value that prevents crashes in the majority of cases.
+		return #if (android) "100" #elseif (web) "100" #else "100" #end;
```

Comment:
added another #elseif for mac and make it 120 because that the only version that is supported and then put else 100

Reply:
make sure to do it for everyone of the code that says 100 even the extension

Reply:
it will look like this 
![Screen Shot 2022-12-02 at 10 23 01 PM](https://user-images.githubusercontent.com/107599365/205427870-a2229495-595f-4612-9bbb-b376b0e06e55.png)
![Sc
![Screen Shot 2022-12-02 at 10 23 35 PM](https://user-images.githubusercontent.com/107599365/205427882-383b6e03-e133-4830-92b7-e27bc5e909de.png)
reen Shot 2022-12-02 at 10 23 21 PM](https://user-images.githubusercontent.com/107599365/205427877-9db26101-4f12-4271-8151-41fc02717b68.png)
![Screen Shot 2022-12-02 at 10 23 50 PM](https://user-images.githubusercontent.com/107599365/205427892-ea79ee98-ba19-4d36-a24d-c8b62b3593f5.png)

Reply:
Wrapping the 100 makes no sense, if you force `100` on a platform that doesn't support it you should obviously get an error.

Thanks for letting me know the correct default version for Mac though.